$OpenBSD: patch-filter_raster_c,v 1.3 2012/08/04 14:37:04 ajacoutot Exp $

XXX push upstream
raster.c:154: error: 'intptr_t' undeclared (first use in this function)

--- filter/raster.c.orig	Fri Oct  7 22:21:45 2011
+++ filter/raster.c	Sat Feb  4 10:36:51 2012
@@ -45,6 +45,7 @@
  * Include necessary headers...
  */
 
+#include <stdint.h>
 #include <cups/raster-private.h>
 
 
